src/app/pages/features/features.component.html

```diff
@@ -93,12 +93,13 @@ <h3>Control your smart devices</h3>
       <h3>Simple user interface</h3>
       <p>
         The <a href="https://github.com/oznu/homebridge-config-ui-x">Homebridge UI</a> provides an easy to use
-        interface to manage your Homebridge plugins, configuration and accessories.
+        interface to manage your Homebridge plugins, configuration, and accessories.
       </p>
       <ul>
         <li>Install and configure Homebridge plugins</li>
-        <li>Monitor your Homebridge server via a fully customisable widget-based dashboard</li>
+        <li>Monitor your Homebridge server via a fully customizable widget-based dashboard</li>
         <li>View and control Homebridge accessories</li>
+        <li>Create child bridge for each Homebridge plugin</li>
```

Comment:
bridges *

Reply:
Which One:
- Create a child bridge for each Homebridge plugin
- Create child bridges for each Homebridge plugin
- Create child bridge(s) for each Homebrige plugin

Reply:
Choose to go with #1.